Keas.Mvc/ClientApp/util/dates.ts

```diff
@@ -9,6 +10,16 @@ export class DateUtil {
         return moment(expiration).format("MM/DD/YYYY").toString();
     }
 
+    public static formatAssignmentExpiration(assignment: IHasExpiration) {
+        if(assignment === null) {
```

Comment:
@jcubbage You can simplify to `if (!assignment || !assignment.expiresAt) {}` but otherwise I like this I think it's a clean way of doing it